assets/js/search.js

```diff
@@ -40,10 +40,8 @@ function locateMe() {
                     response.json().then
```

Comment:
We may also use a good library called axios to simplify fetch API, the syntax will be something like 
`const data = await axios.get(url[, config])`
https://github.com/axios/axios#request-method-aliases

Reply:
Done. Great suggestion 💯